src/test/java/de/bonndan/nivio/output/map/RenderCacheTest.java

```diff
@@ -43,41 +43,48 @@ public void setup() {
 
     @Test
     void toPNGCached() {
-        Landscape landscape = getLandscape("test");
+        Landscape landscape = getLandscape("test", "testLandscape");
         String first = renderCache.getSVG(landscape, false);
         String second = renderCache.getSVG(landscape, false);
 
-        verify(stylesheetFactory, times(1)).getMapStylesheet(any(), any());
+        verify(stylesheetFactory, times(1))
+                .getMapStylesheet(any(), any());
     }
 
     @Test
     void cachesBasedOnIdentifier() {
-        Landscape one = getLandscape("test");
-        String first = renderCache.getSVG(getLandscape("test"), false);
-        Landscape two = getLandscape("test");
+        Landscape one = getLandscape("first", "one");
+        String first = renderCache.getSVG(
+                getLandscape("first", "testLandscape"),
+                false
+        );
+        Landscape two = getLandscape("second", "two");
         two.setProcessLog(one.getLog()); //sync last update
-        two.setIdentifier("second");
         String second = renderCache.getSVG(two, false);
 
-        verify(stylesheetFactory, times(2)).getMapStylesheet(any(), any());
+        verify(stylesheetFactory, times(2))
+                .getMapStylesheet(any(), any());
     }
```

Comment:
If we work towards immutable objects, the identifier must not be set from the outside. Did I understand the Test as it was before and do you think that it is meaningful now too?

Reply:
You're right. It makes no sense changing the identifier.

Reply:
The test is still valid.

Reply:
Thanks :)